public/js/utils/source-editor.js

```diff
@@ -4,10 +4,7 @@ require("codemirror/lib/codemirror.css");
 require("codemirror/mode/javascript/javascript");
 require("codemirror/mode/htmlmixed/htmlmixed");
 require("../lib/codemirror-mozilla.css");
-// enable to get local search to work
-// require("codemirror/addon/search/search");
-// require("codemirror/addon/search/searchcursor");
-// require("codemirror/addon/search/match-highliter");
```

Comment:
true - we actually only need the searchcursor, which should be excluded from the bundle because the source-editor is excluded